mautic/mautic
# Segment email send to email address once

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          |  no
| New feature/enhancement? (use the a.x branch)      |  yes
| Deprecations?                          |  no
| BC breaks? (use the c.x branch)        | no
| Automated tests included?              |  yes 



#### Description:



If a user want to store contacts with same the email address (so create duplicates based on email address on purpose), but avoid send more times to that email address, even If contact's are in segments. For example If more employees use same email address, or family use one email address for parents and children.
This PR added new configuration parameter to enable avoid send segment email to more like one unique email address.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create segment
3. Create 3 contacts with same [email@domain.tld](mailto:email@domain.tld) and add them to segment
4. Go to configuration and enable new options - Send to duplicated just once
5. Send segment email
6. See just one email you receive
